web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -81,44 +79,24 @@ export const ApiHelpers = (pathPrefix, defaultMetricsWindow = '10m') => {
     metricsWindow = window;
   };
 
-  const metricsUrl = `/api/metrics?`;
+  const deploymentUrl = `/api/stat?resource_type=deployment`;
   const urlsForResource = {
-    // all deploys (default), or a given deploy if specified
+    // all deploys (default), or all deploys in a given namespace, or a given
+    // deploy if specified
```

Comment:
since we only ever call `/api/stat?resource_type=deployment`, what do you think about removing support for `namespace` and `resource_name`?